db/schema.rb

```diff
@@ -13,11 +13,10 @@
 ActiveRecord::Schema.define(version: 2022_06_08_035549) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```

Comment:
Also please remove this file from your PR - you didn't change the db for this feature so we shouldn't commit unrelated changes